container_files/public_html/doc/builtin/Url.md

```diff
@@ -63,15 +61,17 @@ Credentials are specified by giving rules.  A credentials rule contains two part
     as the location of that resource and other metadata required to complete the request;
 2.  A rule which tells us which requests are covered by this resource.
 
-When a resource that requires credentials is requested, MLDB will ask the credentials
-daemon for those credentials, passing it the resource and the context (user id, etc)
-in which the request is being made.  The credenials service will scan through its list
-of rules, and return the credentials of the first matching rule to satisfy the request.
+When a resource that requires credentials is requested, MLDB will scan the stored
+rules for those credentials, passing it the resource and the context (user id, etc)
+in which the request is being made.  MLDB will use the first matching rule that satisfies
+the request.
 
 ### Credentials storage
 
-Credentials are stored in the `mldb_data` directory under the `.mldb_credentials` subdirectory.
```

Comment:
Where are the credentials stored on disk?  A user may want to know in order to understand what is stored, debug, or remove credentials.